src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1012,10 +1042,10 @@ private void copySlaveJar(TaskListener listener, String workingDirectory) throws
                         os.close();
                     }
                     listener.getLogger().println(Messages.SSHLauncher_CopiedXXXBytes(getTimestamp(), slaveJar.length));
-                } catch (Exception e) {
+                } catch (Throwable e) {
```

Comment:
You should let `Error` passthrough as is. Escalating to `Throwable` without a passthrough for `Error` is wrong